### ISSUE TITLE
Clang 10 Support?

### ISSUE THREAD

**Issue by nickaleks** [conan-io/conan#5918]
With the release of Clang 9, version 10 is now in development. It is already possible to install it from various llvm repositories (e.g. for Ubuntu). However conan does not seem to support it yet. I have clang-10 on my system, and using conan results in the following (output of conan-cmake):
```
-- Conan: Settings= -pr;default;-s;build_type=Debug;-s;compiler=clang;-s;compiler.version=10;-s;compiler.libcxx=libstdc++11
-- Conan executing: conan install . -pr default -s build_type=Debug -s compiler=clang -s compiler.version=10 -s compiler.libcxx=libstdc++11 -g=cmake -g=virtualrunenv -g=cmake
-- ERROR: Invalid setting '10' is not a valid 'settings.compiler.version' value.
Possible values are ['3.3', '3.4', '3.5', '3.6', '3.7', '3.8', '3.9', '4.0', '5.0', '6.0', '7.0', '7.1', '8', '9']
```

I suppose the fix for this is similar to #3059, but for ordinary clang instead of Apple clang? If yes, I might able to fix this issue, if somebody is not working on this problem already.

**Comment by danimtb** [comment]
Sure, no problem! Thanks for the heads-up

**Comment by memsharded** [comment]
This was closed by #5936
